src/status_im2/contexts/quo_preview/profile/link_card.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im2.contexts.quo-preview.profile.link-card
+  (:require [quo.core :as quo]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:key     :link
+    :type    :select
+    :options [{:key :link}
+              {:key :faceook}
```

Comment:
there's a type here -> face`b`ook.
btw, @ilmotta @smohamedjavid should we namespace these socials keywords like we did with reactions. translations etc? i.e `:socials/facebook`, `:social/github`

Reply:
Agreed @J-Son89, I also prefer that way. It makes searching and refactoring much easier, especially when names can be too common.

I tend to prefer singular names, like `:social/facebook` even though the category _social_ may contain many types.

Reply:
Done